ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -566,6 +560,22 @@ open class DrawerController: UIViewController, FluentUIWindowProvider {
         return dismissPresentingViewController(animated: true)
     }
 
+    func updateCurrentTokens(_ tokens: DrawerTokens) {
+        self.tokens = tokens
+    }
+
+    var tokens: DrawerTokens = .init()
```

Comment:
FYI there's a small update to the tokens branch from #924 that'll require a small change to how this is stored. Check out some of the examples in that PR for guidance, or ping me offline if you have more questions!